test/mongoose_push_web/controllers/dummy_controller_test.exs

```diff
@@ -0,0 +1,13 @@
+defmodule MongoosePushWeb.DummyControllerTest do
```

Comment:
This file uses 4 spaces indentation, while other use double. We should normalize to double-spaces.